src/main/webapp/view/transcript/transcript-area.js

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO: Update this import statement once TranscriptLineElement is in master.
+import {appendTextToList} from '/transcript.js';
+
+/** Loads the transcript lines onto the DOM. */
+export class TranscriptArea {
```

Comment:
Actually one more thing:  Please make this `export default class` because it's the only thing in this file.

Reply:
gotcha, thanks for the info! Done!